lightning/src/offers/invoice.rs

```diff
@@ -135,7 +135,7 @@ use std::time::SystemTime;
 
 pub(crate) const DEFAULT_RELATIVE_EXPIRY: Duration = Duration::from_secs(7200);
 
-/// Tag for the hash function used when signing a [`Bolt12Invoice`]'s merkle root.
+/// Tag for the hash function used when signing a BOLT 12 invoice's merkle root.
```

Comment:
This aligns with the current spec, since static invoices are still officially "invoice" messages. We could add a special exception for static invoices, though, no strong opinion.